### PR TITLE
add a javascript question

### PR DESCRIPTION
**Pull request by ddoyediran**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by ddoyediran** [comment]
@jdwilkin4 Please, let me know if there is anything to add or remove. Thank you!

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,38 +1,52 @@
...
-  Distractor1: "tellig",
-  Distractor2: "ll",
-  Distractor3: "Inigent",
-  Explanation:
-    "The slice() method returns a substring of the original string. The original string remains untouched.",
-  Link: "https://www.freecodecamp.org/news/javascript-substring-examples-slice-substr-and-substring-methods-in-js/",
- },
- {
+  {
+    Question: "Javascript is used for __ side programming?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:3]:
```suggestion
    Question: "What kind of programming can JavaScript be used for?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1,38 +1,52 @@
...
+    Answer: "Both Client and Server",
+    Distractor1: "Server only",
+    Distractor2: "Client only",
+    Distractor3: "None",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:7]:
```suggestion
    Answer: "Both Client and Server side programming",
    Distractor1: "Server side only",
    Distractor2: "Client side only",
    Distractor3: "None of these options",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by ddoyediran** [comment]
Done. Thank you

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**